templates/Notifications/notifications.html.twig

```diff
@@ -0,0 +1,200 @@
+{% extends 'Default/base.html.twig' %}
+
+{% block head %}
+  <link rel="stylesheet" href="{{ asset('css/'~theme()~'/profile.css') }}" media="screen"/>
```

Comment:
what is needed from the profile? maybe it would make more sense to outsource the needed css in its own file to prevent to many unecessary css to be loaded here